docs/source/indexing.rst

```diff
@@ -304,9 +304,6 @@ Database access
 
 After :doc:`configuring <configuration>` one or more database hosts you can access a database with the :py:func:`signac.get_database` function.
```

Comment:
Is this deprecated? If not, we should improve these docs in a separate PR

Reply:
We'll need to do that sometime around the release of signac 2.0. This entire page will probably be removed.